src/Helpers/UtilsDataHelper.php

```diff
@@ -27,10 +34,16 @@ final class UtilsDataHelper
 	 */
 	public static function getDataManifest(string $type, string $file = 'manifest.json'): array
 	{
+		if (isset(self::$esFormsUtilsDataManifests[$type]) && !empty(self::$esFormsUtilsDataManifests[$type])) {
```

Comment:
What is the type of `self::$esFormsUtilsDataManifests[$type]`? Because if you know what type you get out, you can use a better check than `!empty` (`!is_null`, `!is_array`, etc.).

Reply:
I'd opt for the `is_array(self::$esFormsUtilsDataManifests) && isset(self::$esFormsUtilsDataManifests[$type])`. Unless you explicitly want to check for everything `!empty` will return true to https://3v4l.org/ZmlOk

Reply:
well if i add only is array but the array can be empty and I don't want that. I need to check if the key exists and that there is actual data in it.

Reply:
have you considered `if (self::$esFormsUtilsDataManifests[$type] ?? false)`? 😄